### DISCUSSION TITLE
在疫情封闭的上海，告别腾讯 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#7] [General]
# 在疫情封闭的上海，告别腾讯 | 高策

江湖小虾米

http://gaocegege.com/Blog/%E9%9A%8F%E7%AC%94/goodbye-tencent

**Comment by Xuanwo** [comment]
高策哥哥加油！

**Reply by fuyufjh** [reply]
gcgg加油！

**Reply by SunMahatma** [reply]
MIT 6.887 Machine Learning for Systems
http://dsg.csail.mit.edu/6.887/
这个课程感觉挺有意思，内容也都是比较新的东西

**Reply by gaocegege** [reply]
谢谢啦

**Comment by FengJH14** [comment]
祝好～

**Reply by gaocegege** [reply]
:smile: Thanks

**Comment by SunMahatma** [comment]
上面回复错了，抱歉。
MIT 6.887 Machine Learning for Systems
http://dsg.csail.mit.edu/6.887/
这个课程感觉挺有意思，内容也都是比较新的东西

**Comment by willamhou** [comment]
关注好久啦！希望有机会可以一起参与你的项目

**Reply by gaocegege** [reply]
谢谢啦，项目是开源的，到时候会发文章介绍的

**Comment by izackwu** [comment]
高策哥哥加油！

**Reply by gaocegege** [reply]
谢谢树哥！

**Reply by izackwu** [reply]
咱俩各论各的，我叫你高策哥哥，你叫我树哥，不冲突（

**Comment by zhuzilin** [comment]
祝好~
前几天组里的大佬推荐了 wandb，体验真的是很好~

**Reply by gaocegege** [reply]
wandb 可惜就是不是开源的 hh，我感觉它是想做算法工程师的 GitHub

**Comment by huxiaoliang** [comment]
祝你一切顺利，加油

**Reply by gaocegege** [reply]
谢谢 liang 哥！

**Comment by CrownDaisy** [comment]
加油！没问题的。我也在寻找创业的机会。

**Comment by voldikss** [comment]
学长好厉害，预祝创业成功~

**Reply by gaocegege** [reply]
谢谢呀！

**Comment by haiker2011** [comment]
> 通过提供对算法工程师更加友好的构建语言（可以理解为是 Dockerfile 在 AI 场景下的竞品），在更好地处理依赖管理、缓存、Jupyter 与 VSCode remote 等算法工程师常用的 IDE 支持的同时，保证简洁的使用体验。
对于同样在AI平台开发方向的小伙伴，深有同感，`docker commit` 来不停地堆叠依赖也是非常不好的解决方案，但是用户很多时候希望把docker当成虚拟机使用，又是绕不开的功能，支持大神：）

**Reply by gaocegege** [reply]
握手O(∩_∩)O

**Comment by xiaoFine** [comment]
> 所以很多小团队甚至放弃了 Kubernetes、Slurm 等调度系统，选择在公有云上或者自己的小机房内基于裸机开发，通过 Excel 等方式来管理任务。哪个进程先跑起来就算是申请到了资源。
这段太真实了hh
我也一直在做ai-ops相关的工作，期待你的新项目～

**Reply by gaocegege** [reply]
谢谢呀！

**Comment by wissens57** [comment]
6月终于可以外出走走和复工了，不知道大佬这边情况咋样；
总之，祝项目顺利！！！

**Reply by gaocegege** [reply]
https://github.com/tensorchord/envd/ 项目开源了
人也能出去了！

**Reply by wissens57** [reply]
哈哈哈，都是好消息~

**Comment by Marsrocky** [comment]
现在 AI 在越来越多的领域有了落地，算法工程师的数量也与日俱增。

说的太对了，好的工具对交叉学科太重要了！非CS出身的AI研究人员，coding和visualization门槛越低越好。gaocegege加油！

**Reply by gaocegege** [reply]
飞哥也加油 :smile:
